Specs/DateTime/Dutch/TimePeriodExtractor.json

```diff
@@ -269,20 +249,18 @@
         "Length": 11
       }
     ],
-    "NotSupported": "dotNet",
     "NotSupportedByDesign": "javascript,python,java"
   },
   {
-    "Input": "Ik ben weg 20 minuten tot drie voor acht ʼs avonds",
+    "Input": "Ik ben weg 20 minuten voor drie tot acht ʼs avonds",
```

Comment:
To be honest, this case doesn't make any sense to me. I am not sure if I miss something, otherwise I would opt for having this one removed. @LionbridgeCSII @tellarin 

However, if there was already a lot of effort to get it working, we could keep it as an edge case. But in this situation in Dutch, you would say `23 to 8 in the evening` and not `20 before 3 to 8`.

Reply:
If it's plausible and it works, lets keep it.